### ISSUE TITLE
Do not publish new version when upgrading only Gradle

### ISSUE THREAD

**Issue by marcindabrowski** [nicolasfara/conventional-commits#30]
Please do not release a new version when you are not changing anything in the code.
Development dependencies do not introduce new features or bug fixes.

**Comment by nicolasfara** [comment]
Hi @marcindabrowski, what's the problem in this case? Since this project is a Gradle's plugin, should make sense publish a new version as soon as a new Gradle version is released, isn't it?

**Comment by marcindabrowski** [comment]
Hi @nicolasfara I don't agree with you.
The only change you did, was updating Gradle used to build this plugin.
So no new features and bug fixes, so the result will be the same.
You did not start to use the new Gradle API from the merged Gradle update either.

I think we can compare it whit this situation - what is the difference with code compiled with Java 11 update 1 and Java 11 update 2?

**Comment by nicolasfara** [comment]
@marcindabrowski when I started to write plugins for gradle using conventional commits + semantic release I asked myself the question of "when is the right time of publish an update?" and I ended up in this "release policy" for the reason I've explained before. But, thinking about your point of view, you are absolutely right.

I'll fix the release policy in order to publish a new version based on a real change visible for the end users of the plugin.

Thanks for the feedback :)
